arviz/plots/autocorrplot.py

```diff
@@ -29,53 +29,53 @@ def plot_autocorr(
 
     Parameters
     ----------
-    data: obj
+    data : InferenceData
         Any object that can be converted to an :class:`arviz.InferenceData` object
         refer to documentation of :func:`arviz.convert_to_dataset` for details
-    var_names: list of variable names, optional
+    var_names : list of str, optional
         Variables to be plotted, if None all variables are plotted. Prefix the
         variables by ``~`` when you want to exclude them from the plot. Vector-value
         stochastics are handled automatically.
-    filter_vars: {None, "like", "regex"}, optional, default=None
+    filter_vars : {None, "like", "regex"},  default=None
         If `None` (default), interpret var_names as the real variables names. If "like",
         interpret var_names as substrings of the real variables names. If "regex",
         interpret var_names as regular expressions on the real variables names. A la
         ``pandas.filter``.
-    max_lag: int, optional
+    max_lag : int, optional
         Maximum lag to calculate autocorrelation. Defaults to 100 or num draws,
         whichever is smaller.
-    combined: bool, default=False
+    combined : bool, default=False
         Flag for combining multiple chains into a single chain. If False, chains will be
         plotted separately.
     grid : tuple
         Number of rows and columns. Defaults to None, the rows and columns are
         automatically inferred.
-    figsize: tuple
+    figsize : (float, float), optional
         Figure size. If None it will be defined automatically.
         Note this is not used if ``ax`` is supplied.
-    textsize: float
+    textsize : float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
         on ``figsize``.
-    labeller : labeller instance, optional
+    Labeller : Labeller instance, optional
         Class providing the method ``make_label_vert`` to generate the labels in the plot titles.
         Read the :ref:`label_guide` for more details and usage examples.
-    ax: numpy array-like of matplotlib axes or bokeh figures, optional
+    ax : 2D array-like of matplotlib _axes or bokeh _figure, optional
```

Comment:
```suggestion
    ax : 2D array-like of matplotlib_axes or bokeh_figure, optional
```

similar comment. the keywords used here in the parameter type description need to match exactly the alias, so underscore between the library and the class name and no space. They will have a space between the words once rendered with both words being a single link, but not in the raw content here.

Note: these alias aren't defined yet in https://github.com/dameeolawuyi/arviz/blob/apidocs/doc/source/conf.py#L107-L109. You need to update the `conf.py` file following [this other review comment](https://github.com/arviz-devs/arviz/pull/2081/files#r933857288).